### PR TITLE
Initialize examples directory.

### PR DESCRIPTION
**Pull request by MarkDaoust**
This PR pulls files from a few other places, initializing the directory structure Magnus was talking about. 

Into `examples/outreach/blogs`:
* https://github.com/mhyttsten/Misc 

Into `examples/cookbook/regression`:
* https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/get_started/regression

Into `examples/core/get_started`:
* https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/iris.py
* https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/learn/iris_custom_model.py

No files were changed (I renamed the iris files).

I've cleaned-up and added tests for core/get_started examples and cookbook/regression

### CODE REVIEW

**Comment by MarkDaoust** [comment]
Ping.  Any comment here?

The most important thing here is a new home for the `get_started` examples. We'd like to modernize get_started, and it will be a lot less juggling if we can do this move first.

**Comment by martinwicke** [comment]
I'm loving it in principle, I haven't looked at all the code.

File: examples/cookbook/regression/custom_regression.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+# limitations under the License.
+# ==============================================================================
+"""Regression using the DNNRegressor Estimator."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+import tensorflow as tf
```
Comment (nealwu) [inline examples/cookbook/regression/custom_regression.py]: Add a blank line between argparse and tensorflow.

(In general there should be three sections of imports: core Python, e.g., argparse, third-party libraries, e.g., tensorflow, and other local code, e.g., imports85)

File: examples/cookbook/regression/custom_regression.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+import imports85
```
Comment (nealwu) [inline examples/cookbook/regression/custom_regression.py]: `imports85` is a very confusing name. Is there a better name we could use?

File: examples/cookbook/regression/test.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```
Comment (nealwu) [inline examples/cookbook/regression/test.py]: Can we name this file regression_test.py (or regressions_test.py) instead? Our testing script looks for files matching *_test.py.

File: examples/core/get_started/test.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```
Comment (nealwu) [inline examples/core/get_started/test.py]: Same here -- perhaps estimators_test.py

File: examples/outreach/blogs/Blog_Estimators_DataSet.py [hunk 5]
```diff
@@ -0,0 +1,156 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```
Comment (nealwu) [inline examples/outreach/blogs/Blog_Estimators_DataSet.py]: In general file names should be lowercase. Can we rename this?

File: examples/cookbook/regression/custom_regression.py (hunk 1 above)
Comment (MarkDaoust) [reply examples/cookbook/regression/custom_regression.py]: Done.

File: examples/cookbook/regression/custom_regression.py (hunk 2 above)
Comment (MarkDaoust) [reply examples/cookbook/regression/custom_regression.py]: how about "automobile_data" ?

File: examples/cookbook/regression/test.py (hunk 3 above)
Comment (MarkDaoust) [reply examples/cookbook/regression/test.py]: Done

File: examples/core/get_started/test.py (hunk 4 above)
Comment (MarkDaoust) [reply examples/core/get_started/test.py]: Done

File: examples/outreach/blogs/Blog_Estimators_DataSet.py (hunk 5 above)
Comment (MarkDaoust) [reply examples/outreach/blogs/Blog_Estimators_DataSet.py]: Done

**Comment by MarkDaoust** [comment]
I just added a notebook example which we're hoping can support an upcoming blog post. 

Does the repo have a policy on notebooks? This one is 600KB.

**Comment by nealwu** [comment]
We have other notebooks in the repo, e.g., https://github.com/tensorflow/models/blob/master/research/slim/slim_walkthrough.ipynb.

600KB is definitely on the big side though. Is it possible to shrink it? The file and its full history will be downloaded every time anyone clones this repo.

File: examples/cookbook/regression/custom_regression.py (hunk 2 above)
Comment (nealwu) [reply examples/cookbook/regression/custom_regression.py]: "automobile_data" sounds much better.

Comment (nealwu) [reply examples/cookbook/regression/custom_regression.py]: Let's do that in a separate PR to avoid blocking this one.

**Pull request merged**